lib/irb/ext/tracer.rb

```diff
@@ -3,7 +3,8 @@
 #   irb/lib/tracer.rb -
 #   	by Keiju ISHITSUKA(keiju@ruby-lang.org)
 #
-
+# Loading the gem "tracer" will cause it to extend IRB commands with:
+# https://github.com/ruby/tracer/blob/v0.2.2/lib/tracer/irb.rb
```

Comment:
It took me some time to figure this out. Is this helpful?

Reply:
I added that because IRB currently doesn't have integration with it. Do you think dropping it will make things easier here?